Libplanet/Net/Messages/Message.cs

```diff
@@ -341,14 +351,30 @@ private enum MessageFrame
 
             var message = new NetMQMessage();
 
+            // Make signature
```

Comment:
Did you intend this comment for all processes, from building message to make its signature? Or if you intended the comment only for making a signature, it seems more natural to be above on the below lines.

```
            var signature = key.Sign(message.ToByteArray());
            message.Clear();
```

But I think it can be just my personal taste, You can ignore this opinion free if you don't think so.

Reply:
https://github.com/planetarium/libplanet/pull/1181/commits/f3c25c05e3a1a5cbc5215ebc28b9c8adc77b76f8 I fix this comment. please check this out.